### PR TITLE
Bugfix/skills not importing

### PR DESCRIPTION
**Pull request by KathyReid**
* add support for `locale`
* with thanks to @forslund, better support for non-ASCII chars which were prevent imports
* better path handling

### CODE REVIEW

File: automations/extract.py [hunk 1]
```diff
@@ -10,15 +10,31 @@
 btag = "\")"
 
 def main():
+    # add old dialog/en-us, vocab/en-us style paths
+    oldstylepaths = glob("mycroft-skills/**/en-us/**.*", recursive=True)
 
-    pathlist = glob("mycroft-skills/**/en-us/**.*", recursive=True)
+    # add locale path to the pathlist
+    newstylepaths = glob("mycroft-skills/locale/en-us/**.*", recursive=True)
```
Comment (forslund) [inline automations/extract.py]: I don't think this is correct, the path `mycroft-skills/locale/en-us/**.*"` requires the locale folder be directly under the mycroft-skills folder. And it doesn't seem to be necessary:

Whith newstylepaths:
```
 there were 58 LOCALE paths added

 there were 353 DIALOG paths added

 there were 188 VOCAB paths added

 there were 629  paths added in total
```

Without newstylepaths:
```
 there were 58 LOCALE paths added

 there were 353 DIALOG paths added

 there were 188 VOCAB paths added

 there were 629  paths added in total
```

**Comment by KathyReid** [comment]
Thanks @forslund I will take a look and refactor.

**Comment by KathyReid** [comment]
This has been tested OK in `translate-test`

**Comment by forslund** [comment]
Looks good now. Merging

**Pull request merged**